runtime/libc/env.c

```diff
@@ -1,4 +1,4 @@
-/* https://github.com/gwsystems/silverfish/blob/master/runtime/libc/libc_backing.c */
+/* https://github.com/gwsystems/awsm/blob/master/runtime/libc/libc_backing.c */
```

Comment:
Isn’t this a historical link? Will that work?

Reply:
Doesn't resolve. No clue what the point of it is either, so I'll just delete it.